assets/css/privacy.css

```diff
@@ -0,0 +1,37 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300&family=Roboto:wght@300&display=swap');
```

Comment:
Could we consider merging the common styles in this CSS file into the [primary stylesheet](https://github.com/auxfuse/hackathon-git-labs/blob/main/assets/css/style.css) for the project? Adding an additional stylesheet with minimal styles to it for the one page seems a little bit of file directory and separation of concern overkill.

I would recommend creating a new section in the `style.css` file called "Privacy" just before the Media Queries section on line 629.

Once done, please remove this file from the `assets/css` sub-directory and add/commit/push back to the branch associated with this PR to auto-update the PR. 🤜

Reply:
Changes have been made and all added to the `style.css` file. 🤛